staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go

```diff
@@ -3247,6 +3247,9 @@ func describeCertificateSigningRequest(csr *certificatesv1beta1.CertificateSigni
 		w.Write(LEVEL_0, "Annotations:\t%s\n", labels.FormatLabels(csr.Annotations))
 		w.Write(LEVEL_0, "CreationTimestamp:\t%s\n", csr.CreationTimestamp.Time.Format(time.RFC1123Z))
 		w.Write(LEVEL_0, "Requesting User:\t%s\n", csr.Spec.Username)
+		if csr.Spec.SignerName != nil {
+			w.Write(LEVEL_0, "Signer:\t%s\n", *csr.Spec.SignerName)
```

Comment:
also add to pkg/printers/internalversion/printers.go as a new column:
```
	certificateSigningRequestColumnDefinitions := []metav1.TableColumnDefinition{
		{Name: "Name", Type: "string", Format: "name", Description: metav1.ObjectMeta{}.SwaggerDoc()["name"]},
		{Name: "Age", Type: "string", Description: metav1.ObjectMeta{}.SwaggerDoc()["creationTimestamp"]},
		{Name: "Requestor", Type: "string", Description: certificatesv1beta1.CertificateSigningRequestSpec{}.SwaggerDoc()["request"]},
		{Name: "Condition", Type: "string", Description: certificatesv1beta1.CertificateSigningRequestStatus{}.SwaggerDoc()["conditions"]},
	}
	h.TableHandler(certificateSigningRequestColumnDefinitions, printCertificateSigningRequest)
	h.TableHandler(certificateSigningRequestColumnDefinitions, printCertificateSigningRequestList)
```

Reply:
:+1: although worth noting that the internal version does not use a `*string` anymore, so the `if` needs to check for `""` instead (which I think is okay)